docker/docker.go

```diff
@@ -357,9 +374,39 @@ func (d *docker) CollectMetrics(mts []plugin.MetricType) ([]plugin.MetricType, e
 
 // GetConfigPolicy returns plugin config policy
 func (d *docker) GetConfigPolicy() (*cpolicy.ConfigPolicy, error) {
-	return cpolicy.New(), nil
+	policy := pluginlib.NewConfigPolicy()
```

Comment:
@denismakogon totally. If you are going to work on both you can open a second PR, or have 2 commits in this PR - one for each piece - in order to make the review process simple. 👍